ports/libsimpleservo/api/Cargo.toml

```diff
@@ -20,7 +20,7 @@ webxr-api = { git = "https://github.com/servo/webxr", features = ["ipc"] }
 libc = { workspace = true }
 
 [target.'cfg(target_os = "macos")'.dependencies]
-core-foundation = "0.6"
+core-foundation = "0.7"
```

Comment:
Why is this only being updated to 0.7 and not 0.9?

Reply:
Good question. I've updated this change to upgrade it to 0.9. This depends on #30481 though.